### PR TITLE
feat:add me 🐼

### PR DESCRIPTION
**Pull request by Ge6ben**
Hi ! 
All set and ready to go!   🫡
This one’s good to merge, just waiting for your green light.🐼🔥

### CODE REVIEW

**Comment by Ge6ben** [comment]
Hi, Guys Is there anything that should be done from my side ?! @AramRafeq @mhmd-azeez @MiranKm

File: README.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ add other awesome Kurds, including you!
 - [Kawa Idrees](https://github.com/Kawan-edres): Front-end, JavaScript, React, Next.js
 - [Kodo Yousif](https://twitter.com/kodo_koye): React, Node.js Electron, SQL, MongoDB
 - [Kosrat Dlshad](https://github.com/kosratdev): Android, Kotlin, Java, iOS, Swift, TypeScript, Vue
+- [Lara Raoof](https://github.com/Ge6ben): Vue Js, Supabase, AppScript , WebRTC, Charts , GIS and Maps
```
Comment (AramRafeq) [inline README.md:96]: could you change the skills as bellow
`Vue, Supabase, AppScript, WebRTC, Charts, GIS and Maps`

**Review by AramRafeq — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ge6ben) [reply README.md]: Hi @AramRafeq Fixed

**Review by AramRafeq — APPROVED** [review]

**Comment by Ge6ben** [comment]
@AramRafeq  Big Supas for the quick review and approval 🔥🙌🏿

**Comment by Ge6ben** [comment]
@AramRafeq  But why there's no exit from my last commit in the main branch?

**Comment by Ge6ben** [comment]
> [@Ge6ben ](/Ge6ben)
> ` [fix:update skills](/DevelopersTree/awesome-kurds/pull/124/commits/7573c10e0083217d7ebda4fe4b7f3646d9207b57)`

I mean this commit!
https://github.com/DevelopersTree/awesome-kurds/pull/124/commits/7573c10e0083217d7ebda4fe4b7f3646d9207b57

**Comment by AramRafeq** [comment]
I did not understand, your name is on the list with proper skills, i did some.other changes after i merged your PR

**Comment by Ge6ben** [comment]
@AramRafeq Yeah, wired! 
Anyway if it works let's not touch it !!🤭

**Pull request merged**